dandi/consts.py

```diff
@@ -167,7 +167,7 @@ def urls(self) -> Iterator[str]:
 VIDEO_FILE_EXTENSIONS = [".mp4", ".avi", ".wmv", ".mov", ".flv", ".mkv"]
 VIDEO_FILE_MODULES = ["processing", "acquisition"]
 
-ZARR_EXTENSIONS = [".ngff", ".zarr"]
+ZARR_EXTENSIONS = [".ngff", ".zarr", ".nwb"]
```

Comment:
per discussion -- let's try with `.nwb.zarr`.

Reply:
A directory with an `.nwb.zarr` extension has a `.zarr` extension and would thus already be recognized as a Zarr, so there should be no need to modify the code.

Reply:
Very true